### PR TITLE
Add db

### PR DESCRIPTION
**Pull request by tppolkow**
First PR to  just add database connections and migrations.

Database needs to be installed/started seperately from the script.

### CODE REVIEW

File: backend/db/V1_0__create_schema.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
+CREATE SCHEMA IF NOT EXISTS brevity;
+
+CREATE TABLE IF NOT EXISTS brevity.user (
+    user_id INTEGER NOT NULL PRIMARY KEY,
+    openID_id INTEGER,
```
Comment (anthonynd) [inline backend/db/V1_0__create_schema.sql:5]: how does this work? what do people use to log in/sign up?

File: README.md [hunk 2]
```diff
@@ -22,4 +22,7 @@ For anyone looking to run brevity inside the container:
 3. Run `docker image build -t brevity .`
 4. Run `docker run --rm -d --network host --name brevity brevity`
 5. Run `docker exec -it brevity bash`
-6. cd into ~/brevity/ and execute ./run.sh
\ No newline at end of file
+6. cd into ~/brevity/ and execute ./run.sh
+
+## Database
+Database tables should be automatically set up with migrations. Postgresql database needs to be running on your local on port 5432 (the default postgresql port). User and pw should be postgres.
```
Comment (anthonynd) [inline README.md]: how do run the postgres db? is it the command you added to the `run.sh`?

File: backend/db/V1_0__create_schema.sql (hunk 1 above)
Comment (tppolkow) [reply backend/db/V1_0__create_schema.sql:5]: @howard021014  is working on the auth part and we can change this up. But we were planning to just use google auth or something to simplify logins so we don't have to do acc creation and stuff on our app.

File: README.md (hunk 2 above)
Comment (tppolkow) [reply README.md]: No that command just runs the migrations and applies them to your db. The way I installed/run postgres was:

install:
`sudo apt-get install postgresl`

start up:
`sudo su - postgres`

first time only:
`psql`
`\password postgres`
enter new password ( for now just enter postgres )

We could add  the start up line to the run script, but not sure if we want to restart it each time.

**Review by howard021014 — APPROVED** [review]

**Review by AnandMathew — APPROVED** [review]

**Pull request merged**